src/renderer/components/widgets/settings/AddFunds.js

```diff
@@ -196,9 +197,9 @@ export const AddFunds = ({
           >
             Zcash
           </a>{' '}
-          (a cryptocurrency). Cryptocurrency addresses are unique, like email
-          addressess or phone numbers, for money (send to the right address and
-          the recipient will get the money).
+          Cryptocurrencies let you send funds to unique addresses, like email
```

Comment:
@NorbertBodziony I think there's a period missing here before the start of the new sentence, no?

Reply:
Right fixed.